test/controllers/CaptureEmailControllerSpec.scala

```diff
@@ -35,37 +34,40 @@ class CaptureEmailControllerSpec extends ControllerBaseSpec {
   val testValidEmail: String      = "test@example.com"
   val testInvalidEmail: String    = "invalidEmail"
 
-  val mockVatSubscriptionService: VatSubscriptionService = mock[VatSubscriptionService]
-
   val customerInfoResult: CustomerInformation =
-    CustomerInformation(PPOB(
-      PPOBAddress(
-        "address line 1",
-        None,
-        None,
-        None,
-        None,
-        None,
-        "en"
-      ),
-      Some(ContactDetails(
-        None,
-        None,
-        None,
-        Some("test@example.com"),
+    CustomerInformation(
```

Comment:
Can you add in a test to prove the predicate works on the controller? Like how we do for the unauthorised tests? As a sanity check